.storybook/guides/deprecation.mdx

```diff
@@ -58,7 +53,7 @@ At this point you can choose to either immediate move on to the next steps or gi
 Once the deprecation notice has been communicated and the above steps completed, we can remove the component from the codebase safely without breaking local references.
 
 1. Create a new folder with the component name in `.storybook/deprecated`, i.e., `.storybook/deprecated/quickaction`.
-2. Copy `*.stories.js` and `metadata/*.yml` into the new folder (directly, not nested inside subfolders).
+2. Copy `*.stories.js` and `*.mdx` into the new folder (directly, not nested inside subfolders).
```

Comment:
Do we want to add some instructions regarding what to do with the template and test files? I think our approach has changed a little bit, so our markup is all separated, whereas our currently deprecated components have their Template() and Default() all in `*.stories.js`. 

In the future, would we need to copy the template code into `*.stories.js`? Can we safely delete the test file, especially if we're already disabling the Chromatic snapshot?

Reply:
Let's maybe address this in a separate PR?

Reply:
Definitely makes sense!